src/wp-admin/includes/ajax-actions.php

```diff
@@ -3558,7 +3558,7 @@ function wp_ajax_get_revision_diffs() {
 
 	// Removes the script timeout limit by setting it to 0 allowing ample time for diff UI setup.
```

Comment:
```suggestion
```

I think it's better to remove the comment if we are passing a positive value to `set_time_limit()`